aql/src/test/java/org/ehrbase/openehr/sdk/aql/render/AqlRendererTest.java

```diff
@@ -294,6 +302,81 @@ void renderLimit() {
         test(aql, aql);
     }
 
+    @Test
+    void renderVersionedObject() {
+        String aql = "SELECT v/time_created AS first_created " + "FROM EHR e "
+                + "CONTAINS VERSIONED_OBJECT v CONTAINS COMPOSITION";
+
+        test(aql, aql);
+    }
+
+    @Test
+    void renderLatestVersion() {
+        String aql = "SELECT v/commit_audit/time_committed/value AS time_commited, o AS observation "
+                + "FROM EHR e "
+                + "CONTAINS VERSION v[LATEST_VERSION] "
+                + "CONTAINS OBSERVATION o[openEHR-EHR-OBSERVATION.blood_pressure.v2]";
+
+        test(aql, aql);
+    }
+
+    @Test
+    void renderALLVersion() {
+        String aql = "SELECT v/commit_audit/time_committed/value AS time_commited, o AS observation "
+                + "FROM EHR e "
+                + "CONTAINS VERSION v[ALL_VERSIONS] "
+                + "CONTAINS OBSERVATION o[openEHR-EHR-OBSERVATION.blood_pressure.v2]";
+
+        test(aql, aql);
+    }
+
+    @Test
+    void renderPredicateVersion() {
+        String aql = "SELECT v/commit_audit/time_committed/value AS time_commited, o AS observation "
+                + "FROM EHR e "
+                + "CONTAINS VERSION v[commit_audit/time_committed>'2021-12-03T16:05:19.514097+01:00'] "
+                + "CONTAINS OBSERVATION o[openEHR-EHR-OBSERVATION.blood_pressure.v2]";
+
+        test(aql, aql);
+    }
+
+    @Test
+    void renderPredicateTerminology() {
```

Comment:
Why is this called render**Predicate**Terminology?
(also applies to the next method names)